packages/shared/src/normalizeProp.ts

```diff
@@ -29,13 +29,18 @@ export function normalizeStyle(
 
 const listDelimiterRE = /;(?![^(]*\))/g
 const propertyDelimiterRE = /:(.+)/
+const styleCommentRE = /\/\*.*?\*\//g
 
 export function parseStringStyle(cssText: string): NormalizedStyle {
   const ret: NormalizedStyle = {}
   cssText.split(listDelimiterRE).forEach(item => {
     if (item) {
       const tmp = item.split(propertyDelimiterRE)
-      tmp.length > 1 && (ret[tmp[0].trim()] = tmp[1].trim())
+      if (tmp.length > 1) {
+        const key = tmp[0].replace(styleCommentRE, '').trim()
+        const value = tmp[1].replace(styleCommentRE, '').trim()
```

Comment:
Maybe we should replace `cssText` directly as one go?

Reply:
Good idea. Thanks.